Example/SettingsTableViewController.swift

```diff
@@ -162,3 +162,7 @@ class SettingsTableViewController: UITableViewController {
         self.navigationItem.rightBarButtonItem?.isEnabled = true
     }
 }
+
+protocol SettingsTableViewControllerDelegate: class {
+    func settingsTableViewController(_ controller: SettingsTableViewController, updated miniAppList: [MiniAppInfo])
```

Comment:
Maybe a different protocol method name? I feel Protocol name and the method name are much alike

Reply:
that's because the delegate should always have methods recalling its name, like tableView delegate methods for example. Maybe I could simplify it. What do you think ?

Reply:
Yeah, I agree. But I see the protocol name seems to be big and maybe we can have some other prefix like SettingsUpdate/ConfigUpdate or anything.

Reply:
yep, I'll think about it, thanks for noticing this

Reply:
done